lms/djangoapps/certificates/views/support.py

```diff
@@ -208,6 +208,17 @@ def regenerate_certificate_for_user(request):
         )
         return HttpResponseServerError(_("An unexpected error occurred while regenerating certificates."))
 
+    try:
+        # Fetch CertificateInvalidation object
+        certificate_invalidation = CertificateInvalidation.objects.get(
+            generated_certificate=certificate,
+            active=True
+        )
+        # Deactivate certificate invalidation if it was fetched successfully.
+        certificate_invalidation.deactivate()
+    except:  # pylint: disable=bare-except
+        pass
```

Comment:
@awais786 What is the common exception you are catching here? Is it to check whether `certificate_invalidation` exists or not? Isn't that a `CertificateInvalidation. DoesNotExist` exception? Please only catch that and throw others so we don't obfuscate bugs.